discord/app_commands/transformers.py

```diff
@@ -565,7 +572,11 @@ class Range:
         .. code-block:: python3
 
             @app_commands.command()
-            async def range(interaction: discord.Interaction, value: app_commands.Range[int, 10, 12]):
+            async def numeric_range(interaction: discord.Interaction, value: app_commands.Range[int, 10, 12]):
+                await interaction.response.send_message(f'Your value is {value}', ephemeral=True)
+                
+            @app_commands.command()
+            async def string_range(interaction: discord.Interaction, value: app_commands.Range[str, 5, 10]):
```

Comment:
Is this example necessary?

Reply:
Definitely not necessary, but I'd say it's just useful as an additional example.